include/freerdp/constants.h

```diff
@@ -58,6 +58,8 @@ enum RDP_CODEC_ID
 #define OSMINORTYPE_MACINTOSH 0x0006
 #define OSMINORTYPE_NATIVE_XSERVER 0x0007
 #define OSMINORTYPE_PSEUDO_XSERVER 0x0008
-#define OSMINORTYPE_NATIVE_WAYLAND 0x0009
+#define OSMINORTYPE_WINDOWS_RT 0x0009
+/* As of 2022-03-29 the following does not exist officially in [MS-RDPBCGR] */
+#define OSMINORTYPE_NATIVE_WAYLAND 0x000A
```

Comment:
If we start redefining this, it may be a good idea to count down as the range up will (most likely) be successively defined

Reply:
Sure. Choose the value 0xFFFE (if that is okay) (left 0xFFFF as reserved here, in case this would become useful in the future)